dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -462,15 +462,15 @@ ANNO_CLASSSOURCE_FILE_SCAN_EXCEPTION.useraction=See the exception and the associ
 # # {4} The error message from the exception
 ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION=CWWKC0070W: [ {0} ]: The close of resource [{1}] under root [{2}] for class [{3}] failed. Error message: {4}
 ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION.explanation=The close of the file caused an exception.
-ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} A JAR entry name
 # # {2} A class name
 # # {3} The error message from the exception
 ANNO_CLASSSOURCE_JARENTRY_SCAN_EXCEPTION=CWWKC0071W: [ {0} ] The processing of JAR entry [{1}] for class [{2}] caused an exception. Error message: {3}
 ANNO_CLASSSOURCE_JARENTRY_SCAN_EXCEPTION.explanation=The processing of class information read from a JAR entry caused an exception.
-ANNO_CLASSSOURCE_JARENTRY_SCAN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_JARENTRY_SCAN_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.